tests/emp_search.py

```diff
@@ -45,7 +46,27 @@ def test_search_by_job_title(self):
 
         for item in list_of_web_elements_jtitles:
             self.assertEqual('SDET', item.text)
-
+    def test_search_by_employee_id(self):
+        self.login('admin', 'password')
+        self.browser.find_element_by_id('empsearch_id').send_keys('635919')
+        self.browser.find_element_by_id('searchBtn').click()
+        actual_search_result = self.browser.find_element_by_link_text('635919').text
+        print('result' +  actual_search_result)
+        self.assertTrue('635919' == actual_search_result)
+        list_of_web_elements_jtitles = self.browser.find_elements_by_xpath('//*[@id="resultTable"]/tbody/tr/td[5]')
+      #  print("lenght of list is " + str(len(list_of_web_elements_jtitles)))
+        self.assertTrue(len(list_of_web_elements_jtitles)==1)
+    def test_send_esc_key(self):
+        self.login('admin','password')
+        time.sleep(1)
+        self.browser.find_element_by_id('empsearch_employee_name_empName').send_keys('David')
+        self.browser.find_element_by_id('empsearch_employee_name_empName').send_keys(Keys.ESCAPE)
+        self.browser.find_element_by_id('searchBtn').click()
+        list_of_web_elements_jtitles = self.browser.find_elements_by_xpath('//*[@id="resultTable"]/tbody/tr/td[3]')
+        for item in list_of_web_elements_jtitles:
+            self.assertEqual('David', item.text)
+            print("text david" + item.text)
+        time.sleep(4)
```

Comment:
You don't need a sleep here... you may need one after clicking 'Search', but not after the loop :)